src/server/jobs/get_reserves.js

```diff
@@ -97,6 +97,34 @@ const term = args[0];
             deptName: deptNames[i],
             crosslists,
           });
+        } else if (srs === 'Staff') {
```

Comment:
@crystalrain125 This is a big chunk of code duplication from above. I also noticed a bug here and above https://github.com/ucla/lti-library-resources/blob/986cca1f2cb13038dd9afbf3a39678de4d68ca4b/src/server/jobs/get_reserves.js#L70

It is using srs or courseNumber as an index, but those are not unique. SRS is unique when combined with TERM and courseNumber is unique when combined with term and deptName.

Seems this update/log code can be made into a function with parameters to reduce code duplication. Maybe put the function in a model like at /src/server/models/queries.js?

Reply:
@rlorenzo when inserting course with the 'Staff' srs, we don't include a shortname or crosslists. Should I still create 1 function to upsert each db entry in /src/server/models/db.js? I'll fix the bug with the unique query issue.

Reply:
Yeah those can be optional parameters